src/internal/modules/charge-information/controllers/charge-information.js

```diff
@@ -204,12 +204,16 @@ const updateDraftChargeInformation = async (request, h) => {
 
   const preparedChargeInfo = prepareChargeInformation(id, draftChargeInformation);
   preparedChargeInfo.chargeVersion['status'] = 'draft';
-
+  const { user_id: userId, user_name: userName } = get(request, 'defra.user');
```

Comment:
Adding the user details here will overwrite the details of the user who originally submitted the draft charge information.

Reply:
You're right. This was a misunderstanding on my part. I have removed this.